tools/tests/test_cfs_card.py

```
@@ -0,0 +1,366 @@
+
+
+# --------------------------------------------------------------------------- #
+# Behaviour, executed under node
+# --------------------------------------------------------------------------- #
+
+
+@requires_node
+@pytest.mark.parametrize("suite", sorted(p.name for p in JS_TESTS.glob("test_*.mjs")))
+def test_javascript_suite(suite):
+    """Run a node test file and surface its output on failure."""
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)